todo_app/data/trello_items.py

```diff
@@ -1,42 +1,54 @@
-from requests import get, put, post, delete
+import requests
 from todo_app.flask_config import Config
 from todo_app.data.todo_item import TodoItem, TODO_STATUS
 from todo_app.api_exception import ApiException
+import datetime
+
 
 TRELLO_BASE_URL="https://api.trello.com"
 
-config = Config()
+def get_auth_payload(config):
+    return {"key": config.TRELLO_API_KEY, "token": config.TRELLO_API_TOKEN}
+       
 
-auth_payload = {"key": config.TRELLO_API_KEY, "token": config.TRELLO_API_TOKEN}
+def trello_date_time_to_date_time(date_time_str):
+    date_time_obj = datetime.datetime.strptime(date_time_str, '%Y-%m-%dT%H:%M:%S.%fZ')
+    return(date_time_obj)
 
-def get_items():
+def get_items(config):
```

Comment:
Minor, but manually passing this config about everywhere seems a bit unpleasant - are there any better solutions? E.g. we could create a new Config object pretty cheaply whenever it's needed